__tests__/api/health.spec.js

```diff
@@ -16,7 +16,11 @@
 //
 
 import { default as request } from 'supertest'; // eslint-disable-line
-import app from '../../src';
+import expressApp from '../../src';
+import mockAuth from '../../src/libs/mock.authmware';
```

Comment:
Jest's best practices is to put mock objects in a folder called __mocks__ at the same level as the __tests__ folder. Here is an example of how I've mocked `passport` for a different project.
https://github.com/bcgov/devhub-signing-api/blob/master/__mocks__/passport.js